cmd/aws_signing_helper/main.go

```diff
@@ -344,26 +344,25 @@ func main() {
 		var refreshableCred = helper.RefreshableCred{}
 
 		credentialProcessOutput, _ := helper.GenerateCredentials(&credentialsOptions)
-		// buf, _ :=json.Marshal(credentialProcessOutput)
 		refreshableCred.AccessKeyId = credentialProcessOutput.AccessKeyId
 		refreshableCred.SecretAccessKey = credentialProcessOutput.SecretAccessKey
 		refreshableCred.Token = credentialProcessOutput.SessionToken
 		refreshableCred.Expiration, _ = time.Parse(time.RFC3339, credentialProcessOutput.Expiration)
 		endpoint := &helper.Endpoint{PortNum: port, TmpCred: refreshableCred}
 		endpoint.Server = &http.Server{}
 		http.HandleFunc("/", helper.AllIssuesHandlers(&endpoint.TmpCred, &credentialsOptions))
-		//start the credentials endpoint
+		// start the credentials endpoint
 		listener, err := net.Listen("tcp", fmt.Sprintf("%s:%d", helper.Address, endpoint.PortNum))
 		if err != nil {
-			log.Fatal("Fail to create listener\n")
+			log.Println("failed to create listener")
 			syscall.Exit(1)
 		}
 		endpoint.PortNum = listener.Addr().(*net.TCPAddr).Port
 		log.Println("Local server started on port:", endpoint.PortNum)
 		log.Println("Make it available to the sdk by running: ")
-		log.Printf("export AWS_CONTAINER_CREDENTIALS_FULL_URI=http://127.0.0.1:%d", endpoint.PortNum)
+		log.Printf("export AWS_CONTAINER_CREDENTIALS_FULL_URI=http://127.0.0.1:%d\n", endpoint.PortNum)
```

Comment:
Out of curiousity, `\n` is needed here? I know when we are echoing to file, we usually add it for formatting, but don't know if we need it for exporting environment variables. But you may have runned into something I don't know

Reply:
Yeah, it doesn't look like it was needed. I've removed it in the next revision.